packages/@lwc/rollup-plugin/src/__tests__/fixtures/ts_simple_app/src/x/foo/foo.ts

```diff
@@ -0,0 +1,6 @@
+import { api, LightningElement } from "lwc";
+
+export default class Foo extends LightningElement {
+    // @ts-ignore
```

Comment:
why do we need the ignore?

Reply:
Without it my editor will complain "typescript does not support experimental decorators"... I could fix it by adding a tsconfig.json... but that feels like overkill for a simple test so it's easier to just ts-ignore this line